superset-frontend/src/features/databases/CSVUploadModal/index.tsx

```diff
@@ -0,0 +1,805 @@
+                )}
+              </Col>
+            </Row>
+            <Divider orientation="left">{t('Target')}</Divider>
+            <Row justify="space-between">
+              <Col span={11}>
+                <StyledFormItem
+                  label={t('Database')}
+                  name="database"
+                  required
+                  rules={[{ validator: validateDatabase }]}
+                >
+                  <AsyncSelect
+                    ariaLabel={t('Select a database')}
+                    options={loadDatabaseOptions}
+                    onChange={onChangeDatabase}
+                    allowClear
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('Select a database to upload the file to')}
+                </p>
+              </Col>
+              <Col span={11}>
+                <StyledFormItem
+                  label={t('Table Name')}
+                  name="table_name"
+                  required
+                  rules={[
+                    { required: true, message: 'Table name is required' },
+                  ]}
+                >
+                  <Input
+                    aria-label={t('Table Name')}
+                    name="table_name"
+                    data-test="properties-modal-name-input"
+                    type="text"
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('Name of table to be created with CSV file')}
+                </p>
+              </Col>
+            </Row>
+            <Row justify="space-between">
+              <Col span={11}>
+                <StyledFormItem label={t('Schema')} name="schema">
+                  <AsyncSelect
+                    ariaLabel={t('Select a schema')}
+                    options={loadSchemaOptions}
+                    onChange={onChangeSchema}
+                    allowClear
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('Select a schema if the database supports this')}
+                </p>
+              </Col>
+              <Col span={11}>
+                <StyledFormItem label={t('Delimiter')} name="delimiter">
+                  <Select
+                    ariaLabel={t('Choose a delimiter')}
+                    options={delimiterOptions}
+                    onChange={onChangeDelimiter}
+                    allowNewOptions
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('Select a delimiter for this data')}
+                </p>
+              </Col>
+            </Row>
+          </Collapse.Panel>
+          <Collapse.Panel
+            header={
+              <div>
+                <h4>{t('File Settings')}</h4>
+                <p className="helper">
+                  {t('Adjust how the file will be imported.')}
```

Comment:
Suggested text change 
`Adjust how the file will be imported.` -> 
`Adjust how spaces, blank lines, null values are handled and other file wide settings.`